docs/examples/networking/nat/user_data.tpl

```diff
@@ -0,0 +1,11 @@
+write_files:
+  - path: /etc/sysctl.d/98-ip-forward.conf
+    content: |
+      net.ipv4.ip_forward = 1
+
+runcmd:
```

Comment:
Would be useful to add a comment on what this is doing (for those not so familiar with these tools, such as myself).